.github/workflows/pythonapp.yml

```
@@ -35,3 +35,46 @@ jobs:
       run: |
         pip install pytest
         pytest
+    - name: Bump version number
+      if: github.ref == 'refs/heads/master'
+      run: |
+        python -m pip install black
+        echo "NEW_VERSION=$(python .github/workflows/bumper.py)" >> $GITHUB_ENV
+    - name: Commit files
+      if: github.ref == 'refs/heads/master'
+      run: |
+        git config --local user.email "fbda@nfi.nl"
+        git config --local user.name "Version bumper"
+        git add setup.py
+        git commit -m "Bump version to ${{ env.NEW_VERSION }}" -a || echo "Nothing changed in setup.py"
```

Why add `-a` when a single file has just been explicitly added?

---

O, that was a small oversight. Thanks for spotting it

```suggestion
        git commit -m "Bump version to ${{ env.NEW_VERSION }}" || echo "Nothing changed in setup.py"
```